ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/CookiesTest.kt

```diff
@@ -200,5 +200,17 @@ class CookiesTest : ClientLoader() {
         }
     }
 
+    @Test
+    fun testRequestBuilderSingleCookie() = clientTests(listOf("Js")) {
+        test { client ->
+            val result = client.get<String>("$TEST_HOST/respond-single-cookie") {
+                cookies {
```

Comment:
To add some headers to a request one can use the `header` function. Maybe we should stick to the same convention here and use `cookie("single", value = "abacaba")`.

Reply:
It's a good point!

Reply:
But on the contrary, users don't have to know that cookies are implemented via headers, and why would they know that if they are about to send cookies they have to use `headers  {...}` function? 
Is that intuitive?
And btw, on the server side we have : `context.request.header(NAME)` and `context.request.cookies[NAME]` that are separated even though cookies use Cookie header inside. So I would separate these 2 entities on a client for consistency as well.
@e5l What do you think?

Reply:
The server-side request is not a builder. Let's make it similar to the request builder headers

Reply:
I didn't say we need to put the `cookie` function inside the `headers` block.

Reply:
Done